package.json

```diff
@@ -41,7 +41,8 @@
     "federalist": "npx gulp build",
     "lint": "npx gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' ./_site",
+    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --only_4xx --ignore-status-codes 403,429 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' ./_site",
```

Comment:
> [!note]
> This update does two things: 
> - Removes all "ignored" urls to make sure we are still getting 404 status codes for all urls
> - Updates the status codes included in the scan to be all 4xx level status codes, except for 403 and 429. 
> I excluded 403 and 429 because all of the links that html-proofer flagged for these did work during manual testing (For due diligence, I confirmed that every flagged link works for this PR). I am considering them to be non-blocking failures that should be excluded from the circleCI scan. More details about the status code
>   - [403 Forbidden](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403)
>      These include:
>      - Drupal.org links 
>      - Github links, including links to USWDS PRs and issues
>   - [429 Too Many Requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429)
>      These include:
>      - Github links, including links to USWDS PRs and issues

For reference, here is the error report when `ignore-status-codes 403,429` is removed from this script:
```
For the Links > External check, the following failures were found:

* At ./_site/components/data-visualizations/index.html:3559:

  External link https://trends.google.com/trends/explore?date=2020-03-01%202020-03-07&geo=US&q=%2Fm%2F07hyk,%2Fm%2F02yy8 failed (status code 429)

* At ./_site/documentation/implementations/index.html:2817:

  External link https://www.drupal.org/project/uswds_base failed (status code 403)

* At ./_site/documentation/implementations/index.html:2825:

  External link https://www.drupal.org/u/mferanda failed (status code 403)

* At ./_site/documentation/implementations/index.html:2834:

  External link https://www.drupal.org/project/uswds failed (status code 403)

* At ./_site/documentation/implementations/index.html:2842:

  External link https://www.drupal.org/u/brockfanning failed (status code 403)

* At ./_site/documentation/implementations/index.html:2851:

  External link https://www.drupal.org/project/uswds_ckeditor_integration failed (status code 403)

* At ./_site/documentation/implementations/index.html:2859:

  External link https://www.drupal.org/u/smustgrave failed (status code 403)

* At ./_site/documentation/implementations/index.html:2868:

  External link https://www.drupal.org/project/uswds_paragraph_components failed (status code 403)

* At ./_site/documentation/implementations/index.html:2876:

  External link https://www.drupal.org/u/smustgrave failed (status code 403)

* At ./_site/documentation/implementations/index.html:3073:

  External link https://mvnrepository.com/artifact/org.webjars/uswds failed (status code 403)

* At ./_site/documentation/settings/index.html:25342:

  External link https://github.com/uswds/uswds/pull/3876 failed (status code 429)

* At ./_site/documentation/settings/index.html:25396:

  External link https://github.com/uswds/uswds/pull/4059 failed (status code 429)

* At ./_site/documentation/settings/index.html:25414:

  External link https://github.com/uswds/uswds/releases/tag/v2.10.1 failed (status code 429)

* At ./_site/documentation/settings/index.html:25453:

  External link https://github.com/uswds/uswds/pull/3966 failed (status code 429)

* At ./_site/documentation/settings/index.html:25471:

  External link https://github.com/uswds/uswds/releases/tag/v2.10.1 failed (status code 429)

* At ./_site/documentation/settings/index.html:25512:

  External link https://github.com/uswds/uswds/pull/3942 failed (status code 429)

* At ./_site/documentation/settings/index.html:25566:

  External link https://github.com/uswds/uswds/pull/3811 failed (status code 429)

* At ./_site/documentation/showcase/index.html:2666:

  External link https://github.com/uswds/uswds-site/issues/new/choose failed (status code 429)

* At ./_site/documentation/showcase/index.html:2703:

  External link https://pages.18f.gov/digitalaccelerator/ failed (status code 429)

* At ./_site/documentation/showcase/index.html:2723:

  External link https://github.com/presidential-innovation-fellows/fyem-jekyll failed (status code 429)

* At ./_site/documentation/showcase/index.html:2765:

  External link https://www.opioids.gov/ failed (status code 403)

* At ./_site/documentation/showcase/index.html:2775:

  External link https://github.com/GSA/sam-design-system-site failed (status code 429)

* At ./_site/documentation/showcase/index.html:2816:

  External link https://github.com/USDA-FSA/fsa-design-system failed (status code 429)

* At ./_site/documentation/showcase/index.html:2818:

  External link https://github.com/usds/uswds-docs-template failed (status code 429)

* At ./_site/documentation/showcase/index.html:2822:

  External link https://github.com/department-of-veterans-affairs/caseflow failed (status code 429)

* At ./_site/documentation/showcase/index.html:2875:

  External link https://github.com/uswds/uswds-site/pull/2586 failed (status code 429)

* At ./_site/download/index.html:2735:

  External link https://github.com/uswds/uswds-site/pull/2621 failed (status code 429)

* At ./_site/how-to-use-uswds/index.html:2719:

  External link https://github.com/uswds/uswds/wiki failed (status code 429)

* At ./_site/patterns/complete-a-complex-form/establish-trust/index.html:2952:

  External link https://github.com/uswds/uswds-site/pull/1889 failed (status code 429)

* At ./_site/patterns/complete-a-complex-form/keep-a-record/index.html:2826:

  External link https://github.com/uswds/uswds-site/pull/2939 failed (status code 429)

* At ./_site/patterns/complete-a-complex-form/keep-a-record/index.html:2880:

  External link https://github.com/uswds/uswds-site/pull/1889 failed (status code 429)

* At ./_site/patterns/complete-a-complex-form/progress-easily/index.html:2804:

  External link https://github.com/uswds/uswds-site/pull/2744 failed (status code 429)

* At ./_site/patterns/complete-a-complex-form/progress-easily/index.html:2858:

  External link https://github.com/uswds/uswds-site/pull/1889 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3334:

  External link https://www.nytimes.com/2022/06/22/travel/nonbinary-airline-passengers-tickets.html failed (status code 403)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3420:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3473:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3526:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3579:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3629:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3682:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3735:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3788:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/gender-identity-and-sex/index.html:3841:

  External link https://github.com/uswds/uswds-site/pull/1994 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/pronouns/index.html:3071:

  External link https://github.com/uswds/uswds-site/pull/2478 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/race-and-ethnicity/index.html:3221:

  External link https://github.com/uswds/uswds-site/pull/1906 failed (status code 429)

* At ./_site/patterns/create-a-user-profile/race-and-ethnicity/index.html:3274:

  External link https://github.com/uswds/uswds-site/pull/1906 failed (status code 429)

* At ./_site/patterns/select-a-language/selected-content/index.html:2953:

  External link https://github.com/uswds/uswds-site/pull/2566 failed (status code 429)

* At ./_site/patterns/select-a-language/three-or-more-languages/index.html:2949:

  External link https://github.com/uswds/uswds-site/pull/2566 failed (status code 429)

* At ./_site/patterns/select-a-language/two-languages/index.html:2874:

  External link https://github.com/uswds/uswds-site/pull/2566 failed (status code 429)

* At ./_site/performance/glossary/index.html:3010:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/glossary/index.html:3074:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/glossary/index.html:3084:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/glossary/index.html:3089:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/glossary/index.html:3092:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/glossary/index.html:3256:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/glossary/index.html:3261:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/how/index.html:3121:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/how/index.html:3152:

  External link https://www.webpagetest.org/ failed (status code 403)

* At ./_site/performance/why/index.html:2899:

  External link https://www.fastcompany.com/1825005/how-one-second-could-cost-amazon-16-billion-sales failed (status code 403)

* At ./_site/templates/404-page/index.html:4419:

  External link https://github.com/uswds/uswds/pull/5045 failed (status code 429)

* At ./_site/templates/404-page/index.html:4473:

  External link https://github.com/uswds/uswds/pull/4186 failed (status code 429)

* At ./_site/templates/404-page/index.html:4523:

  External link https://github.com/uswds/uswds/pull/4003 failed (status code 429)

* At ./_site/templates/authentication-pages/create-account/index.html:5223:

  External link https://github.com/uswds/uswds/pull/4213 failed (status code 429)

* At ./_site/templates/authentication-pages/create-account/index.html:5277:

  External link https://github.com/uswds/uswds/pull/3843 failed (status code 429)

* At ./_site/templates/authentication-pages/multiple-sign-in-options/index.html:4933:

  External link https://github.com/uswds/uswds/pull/4213 failed (status code 429)

* At ./_site/templates/authentication-pages/multiple-sign-in-options/index.html:4987:

  External link https://github.com/uswds/uswds/pull/3843 failed (status code 429)

* At ./_site/templates/authentication-pages/sign-in/index.html:4602:

  External link https://github.com/uswds/uswds/pull/4213 failed (status code 429)

* At ./_site/templates/authentication-pages/sign-in/index.html:4656:

  External link https://github.com/uswds/uswds/pull/3843 failed (status code 429)

* At ./_site/templates/documentation-page/index.html:3947:

  External link https://github.com/uswds/uswds/pull/5794 failed (status code 429)

* At ./_site/templates/form-templates/password-reset-form/index.html:3000:

  External link https://github.com/usnistgov/800-63-3/blob/nist-pages/sp800-63b/appA_memorized.md failed (status code 429)

* At ./_site/together/index.html:437:

  External link https://www.congress.gov/bill/115th-congress/house-bill/5759/text failed (status code 403)

* At ./_site/utilities/display/index.html:8217:

  External link https://github.com/uswds/uswds/pull/5633 failed (status code 429)

* At ./_site/utilities/display/index.html:8267:

  External link https://github.com/uswds/uswds-site/pull/1630 failed (status code 429)

* At ./_site/utilities/font-size-and-family/index.html:6851:

  External link https://github.com/uswds/uswds/pull/5388 failed (status code 429)

* At ./_site/utilities/height-and-width/index.html:6623:

  External link https://github.com/uswds/uswds/pull/4811 failed (status code 429)

* At ./_site/utilities/height-and-width/index.html:6676:

  External link https://github.com/uswds/uswds-site/pull/1265 failed (status code 429)

* At ./_site/utilities/index.html:5333:

  External link https://github.com/uswds/uswds-site/pull/2731 failed (status code 429)

* At ./_site/utilities/index.html:5390:

  External link https://github.com/uswds/uswds/pull/5209 failed (status code 429)

* At ./_site/utilities/index.html:5510:

  External link https://github.com/uswds/uswds/pull/5079 failed (status code 429)

* At ./_site/utilities/index.html:5718:

  External link https://github.com/uswds/uswds/pull/4198 failed (status code 429)

* At ./_site/utilities/layout-grid/index.html:4148:

  External link https://github.com/uswds/uswds-site/pull/2276 failed (status code 429)

* At ./_site/utilities/layout-grid/index.html:4250:

  External link https://github.com/uswds/uswds-site/pull/1361 failed (status code 429)

* At ./_site/utilities/margin-and-padding/index.html:14393:

  External link https://github.com/uswds/uswds-site/pull/1940 failed (status code 429)

* At ./_site/utilities/margin-and-padding/index.html:14443:

  External link https://github.com/uswds/uswds-site/pull/1940 failed (status code 429)

* At ./_site/utilities/margin-and-padding/index.html:14497:

  External link https://github.com/uswds/uswds/pull/4212 failed (status code 429)

* At ./_site/utilities/text-styles/index.html:5216:

  External link https://github.com/uswds/uswds-site/pull/1214 failed (status code 429)

* At ./_site/website-standards/index.html:2737:

  External link https://github.com/uswds/uswds-site/pull/2843 failed (status code 429)


HTML-Proofer found 87 failures!
```